makefile

```diff
@@ -99,9 +99,9 @@ LIBARCTIC := -L $(DIR_ROOT) -Wl,-rpath,$(DIR_ROOT) -l$(TARGET)
 # Add multithreading to reduce runtime (requires OpenMP to have been installed)
 CXXFLAGS += -Xpreprocessor -fopenmp
 # Use this on a mac
-# LIBS += -L $(DIR_OMP)/lib -lomp
+ LIBS += -L $(DIR_OMP)/lib -lomp

```

Comment:
We need to stop swapping these out depending on whether the last PR was someone using Mac or Linux lol